README.md

```diff
@@ -83,7 +83,16 @@ The lint task will check to see that SVG images are optimized. To optimize image
 npm run optimize-assets
 ```
 
-To build **Help Center** pages, follow the guidelines in the [Help README](_help/README.md).
+### NetlifyCMS
+[NetlifyCMS](https://www.netlifycms.org/) is an open source content management system that we use to edit content on the brochure site. To develop and make changes to the CMS, or to edit content locally, first comment out lines 3-10 in `admin/config.yml`
+and uncomment lines 19-23. After you uncomment those lines, change line 21 to the name of the branch to which you are working on.
```

Comment:
I worry that specific line numbers are also at risk of getting out of date, I'd maybe say "just comment out the `backend:` block and uncomment the `backend:` block that contains a proxy_rul

But same dea